decidim-admin/app/views/layouts/decidim/admin/_header.html.erb

```diff
@@ -3,4 +3,5 @@
 <%= stylesheet_pack_tag "decidim_admin", media: "all" %>
 <%= javascript_pack_tag "decidim_admin", defer: false %>
 <%= organization_colors %>
-<%= snippets.display(:head) %>
+<%= snippets.display(:snippet_styles) %>
```

Comment:
I would suggest against changing the snippet position name here.

If external modules are using the `:head` snippet position, this would break those modules.

I also find it clearer that we call the snippet position `:head` which tells me directly where this snippet position is displayed. When the snippet position is named `:snippet_scripts`, it doesn't tell me where that goes in the DOM.

Also, why do you want to add the `:snippet_` prefix to these? Aren't they already snippets as per the helper name?

Reply:
Agreed